bindings/wasm/package.json

```diff
@@ -74,13 +74,14 @@
     "wasm-pack": "0.10.1"
   },
   "dependencies": {
+    "@iota/iota-client-wasm": "^0.5.0-alpha.6",
```

Comment:
we should remove it from the dependencies

Reply:
Why?